src/components/pages/Department/Department.tsx

```diff
@@ -1,42 +1,66 @@
 import React, { useEffect, useState, useMemo } from 'react';
 import { useParams } from 'react-router-dom';
+import {
+  Button,
+  Col,
+  Container,
+  Row,
+} from 'reactstrap';
 
 import DepartmentsAPI from '../../../API/Departments';
 import { IEmployee } from '../../../types/employees';
+import { IDepartment } from '../../../types/departments';
+import './Department.scss';
+import AddButton from '../../AddButton/AddButton';
+import EditButton from '../../EditButton/EditButton';
 
 const Department: React.FC = () => {
   const { id } = useParams();
   const [employees, setEmployees] = useState<IEmployee[]>([]);
+  const [department, setDepartment] = useState<IDepartment>();
 
   const getDepartmentInfo = async () => {
     try {
       const res = await DepartmentsAPI.getDepartmentInfo(id!);
-
-      setEmployees(res.data.employees);
+      setDepartment(res.data);
+      setEmployees(res.data.employees!);
```

Comment:
No need to set employees to the component state separately rather create a separate component with employee as prop